src/createServer.js

```diff
@@ -1,3 +1,63 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const caseName = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const createServer = () => http.createServer((req, res) => {
+  const errors = [];
+
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const originalText = normalizedURL.pathname.slice(1);
+  const targetCase = normalizedURL.searchParams.get('toCase') || '';
+
+  res.setHeader('Content-type', 'application/json');
+
+  if (!originalText) {
+    errors.push({
+      message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".'
+    });
+  }
+
+  if (!targetCase) {
+    errors.push({
+      message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".'
```

Comment:
You can create an object with all your default error messages and use them as reference